inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/gather_ref.cl

```diff
@@ -5,7 +5,19 @@
 #include "include/data_types.cl"
 #include "include/fetch_data.cl"
 
-#define INPUT_AXIS_INDEX (uint)indices[indices_idx]
+#ifdef INDEX_DIM
+inline const uint GET_POSITIVE_INDEX(int in)
```

Comment:
It's better to use `FUNC()` wrapper for function definition and `FUNC_CALL()` macros for function call to avoid compilation errors, such as "function redefinition error"